src/widget/core.rs

```diff
@@ -446,28 +448,26 @@ impl Pod {
         };
 
         if needs_paint {
-            let mut builder = SceneBuilder::for_fragment(&mut self.fragment);
-            self.widget.paint(&mut inner_cx, &mut builder);
+            self.widget.paint(&mut inner_cx, &mut self.fragment);
```

Comment:
ditto, `self.fragment.reset()`

Reply:
Yeah I noticed quickly. Even after closing the examples my GPU driver crashed my system about 15 seconds later... I missed it in core and made the same mistake. :grimacing: 
Anyway, resolved.